.eslintrc.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     ],
     'react/jsx-props-no-spreading': 'off',
     'import/no-unresolved': [0, { caseSensitive: false }],
+    'no-alert': 'off',
```

Comment:
Instead of putting this, I think we can follow the rules of es-lint
https://docs.w3cub.com/eslint/rules/no-alert
What say ??

Reply:
Hey @rohan09-raj, couldn't find any exception to apply in the rule **no-alert**. Can you please guide how to solve the issue?

Reply:
@Debarshi95 Sorry, I thought warnings can be removed by following the correct rules. I exactly do not know how it can be done, I think we can go with this rule.
Also, please add `'no-console': 'off' and put line 33 back in `fetch.js` which you have removed.
https://github.com/Real-Dev-Squad/website-members/pull/295/files/36ac38a3246fc8063f15702001ba7e9033487225#r778588984

Reply:
why is this required? can't we remove the alert's from the code?

Reply:
@sumitd94 Yes, we can remove it, I forgot that Next JS already has features for popping up errors.

Reply:
@rohan09-raj May be we can show some toast notification or some kind of modal instead of showing alert?

Reply:
I think @sumitd94 is the better person to suggest some good alternatives.

Reply:
Let's leave it for now.
Will create toast service and track it in a different ticket